sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/implementation/FormRecognizerClientImplBuilder.java

```diff
@@ -123,30 +141,49 @@ public FormRecognizerClientImplBuilder httpClient(HttpClient httpClient) {
      * The configuration store that is used during construction of the service
      * client.
      */
-    private Configuration configuration;
+    @Generated private Configuration configuration;
```

Comment:
Is this expected @Generated ?

Reply:
Yeah, this is from another change in Autorest that happened a few months ago.

Reply:
https://github.com/Azure/autorest.java/releases/tag/v4.0.40